tests/tests-tax.php

```diff
@@ -124,6 +125,10 @@ public function test_get_tax_rate() {
 		$this->assertEquals( '0.036', edd_get_tax_rate( 'HK' ) );
 	}
 
+	public function test_get_tax_rate_less_than_one() {
```

Comment:
To be extra thorough, let's get at least 5 different tests added for 5 different rates.